airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/steps/gradle.py

```diff
@@ -249,20 +256,26 @@ async def _collect_test_results(self, gradle_container: Container) -> Optional[A
         Only the XML files generated by junit are downloaded into the host filesystem
         One can change the destination directory by setting the outputs
         """
-        test_results_dir_name = "test-results"
-        if test_results_dir_name not in await gradle_container.directory(f"{self.context.connector.code_directory}/build").entries():
-            self.context.logger.warn(f"No {test_results_dir_name} found directory in the build folder")
+        test_results_dir_name_in_container = "test-results"
+        test_results_dir_name_in_zip = f"test-results-{datetime.fromtimestamp(cast(float, self.context.pipeline_start_timestamp)).isoformat()}-{self.context.git_branch}-{self.gradle_task_name}".replace(
```

Comment:
nit: this can be declared after the next `if`statement as its only used if the `test_results_dir_name_in_container`exists.